package/MDAnalysis/analysis/hole.py

```diff
@@ -258,6 +257,10 @@
 import logging
 from itertools import cycle
 
+import numpy as np
+import matplotlib
```

Comment:
yes, but we don't use it a lot and it's clearer to read and to grep for: if I am looking for matplotlib I use `git grep matplotlib`.